<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceCreationDialog</class>
 <widget class="QDialog" name="DeviceCreationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Logical caracteristics</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="vendorComboBox"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Vendor</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="typeComboBox"/>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="locationComboBox"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Location</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Instance</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="instanceLinEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Zigbus Caracteristics</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Module</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="moduleComboBox"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Pin Type</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QComboBox" name="pinTypeComboBox"/>
      </item>
      <item row="1" column="0" rowspan="2" colspan="4">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Main pin</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="1">
          <widget class="QRadioButton" name="pin1RadioNumeric">
           <property name="text">
            <string>numeric</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="pin1RadioAnalogic">
           <property name="text">
            <string>analogic</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="2">
          <widget class="QComboBox" name="pin1ComboBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" colspan="4">
       <widget class="QGroupBox" name="optionPinGroupBox">
        <property name="title">
         <string>Optional pin</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0" rowspan="4">
          <widget class="QComboBox" name="pin2ComboBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1" rowspan="2">
          <widget class="QRadioButton" name="pin2RadioNumeric">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>numeric</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" rowspan="2">
          <widget class="QRadioButton" name="pin2RadioAnalogic">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>analogic</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Pin SubType</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="2">
       <widget class="QComboBox" name="pinSubTypeComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DeviceCreationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
